### DISCUSSION TITLE
Perform general sequence classification

### DISCUSSION THREAD

**Discussion by marcmk6** [autogluon/autogluon#1598] [Q&A]
Hi,

I'm wondering if it's possible to train a sequence classifier (for non-natural language sequences) from random initialization, instead of doing transfer learning on pre-trained natural language models. 

Thanks

**Comment by Innixma** [comment]
One way is to simply pass your data to TabularPredictor as a supervised learning problem. More advanced options are not currently supported, however we do plan to released time-series support in v0.5, which may suit your needs.

**Reply by marcmk6** [reply]
Thanks for responding!
Let me clarify. I'm still dealing with sequences of discrete tokens, rather than continuous time-series data.
What I want to know is if there's a way to do a very quick prototyping of a sequence classifier with just a few lines of code, given a bunch of discrete token sequences (**not in human language**) and corresponding labels. 

(The built-in TextPredictor seems to be Transformer-based models, but are supposed to be fine-tuned from pre-trained weights. I still want to use these model architectures as well as all the amazing automated stuff provided by AutoGluon, but random model initialization.)

**Reply by Innixma** [reply]
@sxjscience is this supported in AutoMM / Text?
